site/daemon.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Weave Flux Daemon
+menu_order: 90
+---
+
+# Weave Cloud Deploy
+
+Flux daemon (fluxd, aka flux agent) is at the heart of Weave Cloud Deploy, functionality allowing automation of application deployments and version control of cluster configuration.
+Version controlling of cluster manifests provides reproducibility and a historical trail of events.
+
+## Flux daemon responsibilities
+
+    A) Continuous Deployment
+	    1.
+	    Flux daemon monitors container registry for running container image updates.
+        Detection of an image change (running container image tag vs container
+        registry image tag) triggers k8s manifest update, which is committed to the
+        user git repository, then deployed to the Kubernetes cluster.
+
+	    2.
```

Comment:
Can we switch these two around -- sync first, automation second (just a cosmetic thing)

Reply:
Done.